src/quo2/components/community/divider_label.cljs

```diff
@@ -0,0 +1,61 @@
+(ns quo2.components.community.divider-label
```

Comment:
this component should be in `dividers` 
i.e 
quo2.components.dividers.divider-label

It's used in the communities designs but we have agreed to the keep the quo2 library structure matching the Figma designs structure - https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Mobile-Components?node-id=1338%3A34482